packages/model-viewer/src/test/features/scene-graph/model-spec.ts

```diff
@@ -186,6 +186,25 @@ suite('scene-graph/model', () => {
                   element[$scene].width, element[$scene].height)!;
               expect(material).to.be.null;
             });
+        test('uvFromPoint returns texture-coordinates', async () => {
+          await loadModel(ASTRONAUT_GLB_PATH);
+
+          const uv = element.uvFromPoint(
+              element[$scene].width / 2, element[$scene].height / 2)!;
+
+          expect(uv).to.be.ok;
+        });
+
+        test(
+            'uvFromPoint returns null when intersect fails', async () => {
+              await loadModel(ASTRONAUT_GLB_PATH);
+
+              await timePasses(1000);
```

Comment:
Is this necessary?

Reply:
Here I was following the test from "materialFromPoint()" for coherence. I removed it, and just check that the UV-coords are within 0-1 inside the positionAndNormalFromPoint() test.